packages/maskbook/src/plugins/ITO/SNSAdaptor/hooks/useSpaceStationClaimableTokenCountCallback.ts

```diff
@@ -0,0 +1,23 @@
+import { getClaimableTokenCount } from '../../Worker/apis/spaceStationUUPS'
+import { useCallback, useState } from 'react'
+import type { ClaimableCount } from '../../types'
+
+export function useSpaceStationClaimableTokenCountCallback() {
+    const [spaceStationAccountClaimableLoading, setSpaceStationAccountClaimableLoading] = useState(false)
+    const [spaceStationClaimableCount, setSpaceStationClaimableCount] = useState<ClaimableCount>()
+    const spaceStationAccountClaimableCallback = useCallback(
+        async (address: string) => {
+            setSpaceStationAccountClaimableLoading(true)
+            const data = await getClaimableTokenCount(address)
```

Comment:
Are you sure calling those APIs without using RPC?

Reply:
yes